libcxx/include/module.modulemap.in

```diff
@@ -448,10 +448,6 @@ module std_inttypes_h [system] {
   export *
 }
 // <iso646.h> provided by compiler.
-module std_limits_h [system] {
```

Comment:
Whey this removed, because `stdint.h` are also provided by compiler

Reply:
Yes, if it is provided by the compiler then the compiler should provide an appropriate modulemap for that header if that's necessary.